rpcs3/Emu/RSX/VK/VKFormats.cpp

```diff
@@ -39,6 +39,32 @@ VkFormat get_compatible_depth_surface_format(const gpu_formats_support &support,
 	throw EXCEPTION("Invalid format (0x%x)", format);
 }
 
+std::tuple<VkFilter, VkSamplerMipmapMode> get_min_filter_and_mip(u8 min_filter)
+{
+	switch (min_filter)
+	{
+	case CELL_GCM_TEXTURE_NEAREST: return std::make_tuple(VK_FILTER_NEAREST, VK_SAMPLER_MIPMAP_MODE_NEAREST);
```

Comment:
Btw, is there any reason to use tuples instead of pair for two elements? I saw similar changes in your other PRs. Tuples are potentially less efficient, at least without full optimizations.